assets/css/app.css

```diff
@@ -229,4 +229,10 @@ input:-webkit-autofill:focus {
   unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
   font-weight: 500;
   src: local("AvertaStd-Semibold"), local("Averta Std Semibold"), url(https://sportsbet.io/sportsbet-io/files/fonts/averta-std/semibold.woff2) format("woff2");
-}
\ No newline at end of file
+}
+
+@layer components {
```

Comment:
should this remain here? there is no sidebar now

Reply:
good notice!